io/file/binary writer/a92a6e2b-166e-4b43-8dc4-3674d8ef5faf.json

```diff
@@ -4,21 +4,25 @@
     "inputs": {
       "2f960cd2-ce95-47dd-9dfa-7b89eefe48e3": {
         "type": "string",
-        "name": "path"
+        "name": "path",
+        "description": "Receives the path of the generated binary file.\n\nExample:\n\"c:/image.png\"@1"
       },
       "4b68e52a-3d8d-4db6-857c-ae1448dd7eb1": {
         "type": "string",
-        "name": "base64"
+        "name": "base64",
+        "description": "Receives the base64 string to be written as binary file.\n\nExample:\n\"VGhlIGJhc2tldCBpcyBmdWxsIG9mIGdyYXBlcy4=\"@1"
       }
     },
     "outputs": {
       "0cbefbca-31bc-4056-8b5a-5bf384e91f94": {
         "type": "null",
-        "name": "written"
+        "name": "written",
+        "description": "Sends null value if the file writting was successful.\n\nExample:\nnull@1"
```

Comment:
I'd consider sending out the path of the successfully written file on this port instead of null.
Then you could continue to use this port as an event trigger just like with `null` but you could also chain other nodes on it that would use the path for other purposes simplifying thus the flow.
I know it is a kind of breaking change but as we don't have type matching yet it wouldn't cause any harm in existing projects.

Reply:
File deleter, File copier nodes already use the above mentioned output schema

Reply:
Done